script.js

```diff
@@ -20,79 +22,55 @@ instance
       inprogressStories,
       doneStories
     );
-
     displayStories(todoStories, reviewStories, inprogressStories, doneStories);
+    initializeDragAndDrop();
   })
   .catch(function(error) {
     console.log(error);
   });
-
-function organizeStories(
-  allStories,
-  todoStories,
-  reviewStories,
-  inprogressStories,
-  doneStories
-) {
-  for (let i = 0; i < allStories.length; i++) {
-    if (
-      allStories[i].current_state === "unscheduled" ||
-      allStories[i].current_state === "unstarted"
-    )
+  
+  function organizeStories(
+    allStories,
+    todoStories,
+    reviewStories,
+    inprogressStories,
+    doneStories
+  ) {
+    for (let i = 0; i < allStories.length; i++) {
+      if (
+        allStories[i].current_state === "unscheduled" ||
+        allStories[i].current_state === "unstarted"
+      )
       todoStories.push(allStories[i]);
-    if (
-      allStories[i].current_state === "started" ||
-      allStories[i].current_state === "rejected" 
-    ) {
-      inprogressStories.push(allStories[i]);
+      if (
+        allStories[i].current_state === "started" ||
+        allStories[i].current_state === "rejected" 
+      ) {
+        inprogressStories.push(allStories[i]);
+      }
+      
+      if (allStories[i].current_state === "finished") {
+        reviewStories.push(allStories[i]);
+      }
+      if (
+        allStories[i].current_state === "delivered" ||
+        allStories[i].current_state === "accepted"
+      )
+      doneStories.push(allStories[i]);
     }
-
-    if (allStories[i].current_state === "finished") {
-      reviewStories.push(allStories[i]);
+  }
+  
+  function displayStories(todoStories, reviewStories, inprogressStories, doneStories) {
```

Comment:
I believe you can use the `rest args` feature here (might need to check browser compatibility):

```
function displayStories(...storyColumns) {
```

Then you can just call `storyColumns.forEach()` in the function.